src/DynamoMigratorExtension.cs

```diff
@@ -35,20 +36,45 @@ public string Name
             }
         }
 
-        private string selectedDirectory = "No directory selected";
+        private string selectedSourceDirectory = "No directory selected";
         /// <summary>
-        /// The selected directory of dynamo files to be migrated.
+        /// The selected source directory of dynamo files to be migrated.
         /// </summary>
-        public string SelectedDirectory
+        public string SelectedSourceDirectory
         {
             get
             {
-                return this.selectedDirectory;
+                return this.selectedSourceDirectory;
             }
             set
             {
-                selectedDirectory = value;
-                RaisePropertyChanged(nameof(SelectedDirectory));
+                selectedSourceDirectory = value;
+                RaisePropertyChanged(nameof(SelectedSourceDirectory));
+            }
+        }
+
+        public string selectedTargetDirectoryDefault
+        {
+            get
+            {
+                return "No target directory selected, files will be overwritten in source directory.";
+            }
+        }
+
+        private string selectedTargetDirectory = "No target directory selected, files will be overwritten in source directory.";
```

Comment:
what do you think about doing
`selectedTargetDirectory = selectedTargetDirectoryDefault;`

Reply:
@mjkkirschner that was actually my initial intention, but it gave me an error in VS:
```A field initializer cannot reference the non-static field, method, or property```
So that would mean I'd have to declare selectedTargetDirectoryDefault as static? My albeit very limited knowledge of C# tells me I would have to do that inside a constructor (which the class currently doesn't have) - would that be okay or is there something I'm missing? Not sure how to that in a class that doesn't have a constructor, but my C# skills are quite limited...

Reply:
so I don't see a reason this member (selectedTargetDirectoryDefault) really needs to be a property with a public getter, lets just make this simple and make it a public const string - that should act like a public static field. If you can't get it working we can leave it like this for now.

Reply:
![grafik](https://user-images.githubusercontent.com/3014437/37849055-8a1887f0-2ed6-11e8-84f8-6b20abdbd658.png)

Reply:
So that works well within DynamoMigrationExtension.cs but I need to compare the selected target directory with the default value in migrationWindow.xaml.cs as well. There I get a CS1076 error (```cannot be accessed with an instance reference, qualify it with a type name instead```) but when I call it as DynamoMigratorExtension.selectedTargetDirectoryDefault instead, VS starts yapping about the implementation of Dispose (CA 1063).

Reply:
I think thats just a warning - go ahead and make the change if it builds and I'll check it out in my next PR.